docs/design/service_traffic_policy.md

```diff
@@ -315,3 +315,134 @@ where packet leaves the node and goes back to the external entity that initiated
 
 ## Sources
 - https://www.asykim.com/blog/deep-dive-into-kubernetes-external-traffic-policies
+
+## Internal Traffic Policy
+
+Service Internal Traffic Policy (ITP) is a feature that can be enabled on a kubernetes service type object. This feature imposes restrictions on traffic that originates internally by routing it only to endpoints that are local to the node from where the traffic originated from. Here "internal" traffic means traffic originating from pods and nodes within the cluster. See https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy/ for more details.
+
+```
+NOTE1: ITP is applicable only to service of type "ClusterIP" meaning it has no effect on services of type NodePorts, ExternalIPs or LoadBalancers. So if
```

Comment:
This isn't entirely accurate -- it's not the service type, but the traffic type. It *might* be the case that internal traffic to an ExternalIP is treated as internal (though I'm not sure).

Reply:
yeah I tried to add that second sentence to make it sound clearer :D but probably traffic type sounds better.

I confirmed with Dan Winship that only internal traffic to clusterIP is treated internal, externalIPs and LBs aren't: https://docs.google.com/document/d/12rx3ix9jpmneBZGbBKs6c1caEKhSKLfwODQ-_CN8hmQ/edit#bookmark=id.h9bkqn3sh4ex